docs/content/development/datasketches-aggregators.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: doc_page
+---
+
+## DataSketches aggregator
+Druid aggregators based on [datasketches]()http://datasketches.github.io/) library. Note that sketch algorithms are approxiate, see details in the [datasketches doc](http://datasketches.github.io/docs/theChallenge.html).
```

Comment:
I think a description of high level when to use the aggregators and post aggregators is required